main.go

```diff
@@ -296,6 +297,23 @@ func collectFilesToDeploy(absDeployPth string, config Config, tmpDir string) (fi
 	return filesToDeploy, nil
 }
 
+func compressAsTar(sourceDirPath, destinationTarPath string, isContentOnly bool) error {
+	fmt.Println()
+	log.Infof("Compressing directory...")
+
+	// -c - create a new archive
+	// -f - the next argument is the name of the output archive
+	// Note: recursive behavior is default in tar
+	cmd := command.New("/usr/bin/tar", "cf", destinationTarPath, sourceDirPath)
```

Comment:
I don't think it's a good idea to use an absolute path here, we can't make sure it's the same on all systems. For example, on M1 macOS, `/usr/bin` is read-only, so any manually installed `tar` will live in a different directory (such as `/opt/homebrew/bin`).